syntax = "proto3";

message Contact {
    oneof a_user_id {
        int32 a_user_id_int = 1;
        string a_user_id_string = 2;
    }
    oneof b_user_id {
        int32 b_user_id_int = 3;
        string b_user_id_string = 4;
    }

    string a_user_name = 5;
    string b_user_name = 6;
}
